cmd/cnab-run/bundle.go

```diff
@@ -2,27 +2,27 @@ package main
 
 import (
 	"github.com/deislabs/cnab-go/bundle"
-	"github.com/docker/app/internal/relocated"
+	"github.com/docker/app/internal/image"
 	"github.com/docker/cnab-to-oci/relocation"
 )
 
 const (
-	// bundlePath is where the CNAB runtime will put the actual Bundle definition
+	// bundlePath is where the CNAB runtime will put the actual AppImage definition
 	bundlePath = "/cnab/bundle.json"
 	// relocationMapPath is where the CNAB runtime will put the relocation map
 	// See https://github.com/cnabio/cnab-spec/blob/master/103-bundle-runtime.md#image-relocation
 	relocationMapPath = "/cnab/app/relocation-mapping.json"
 )
 
 func getBundle() (*bundle.Bundle, error) {
-	return relocated.BundleJSON(bundlePath)
+	return image.BundleJSON(bundlePath)
 }
 
 func getRelocationMap() (relocation.ImageRelocationMap, error) {
-	return relocated.RelocationMapJSON(relocationMapPath)
+	return image.RelocationMapJSON(relocationMapPath)
 }
 
-func getRelocatedBundle() (*relocated.Bundle, error) {
+func getRelocatedBundle() (*image.AppImage, error) {
```

Comment:
super nit: What about `image.App` ? `image.AppImage` sounds like repetition

Reply:
Better rename package then. "App" is highly confusing with "running app".